### ISSUE TITLE
[bug] dev-tool 3.4.4 can not create project

### ISSUE THREAD

**Issue by zhangmazi** [phalcon/phalcon-devtools#1329]
### Details
Environment:
  OS: Darwin rickmac.local 18.7.0 Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64 x86_64
  PHP Version: 7.2.23
  PHP SAPI: cli
  PHP Bin: /usr/local/Cellar/php@7.2/7.2.23/bin/php
  PHP Extension Dir: /usr/local/Cellar/php@7.2/7.2.23/lib/php/20170718
  PHP Bin Dir: /usr/local/Cellar/php@7.2/7.2.23/bin
  Loaded PHP config: /usr/local/etc/php/7.2/php.ini
Versions:
  Phalcon DevTools Version: 3.4.0
  Phalcon Version: 3.4.4
  AdminLTE Version: 2.3.6

run code
```
phalcon project a123
```

error info
```
PHP Warning:  mkdir(): Permission denied in /Users/xxxxx/works/web/github/phalcon-devtools/scripts/Phalcon/Builder/Project.php on line 105

Warning: mkdir(): Permission denied in /Users/xxxxx/works/web/github/phalcon-devtools/scripts/Phalcon/Builder/Project.php on line 105

 Error: Unable create project directory /a123/
```

**Comment by Jeckerson** [comment]
Try to create **not** on the root of your file system. AFAIK MacOS doesn't like that.
Normally you create all stuff inside your home dir
```
cd ~
phalcon project a123
```

**Comment by KozakIvan** [comment]
# phalcon i

Phalcon DevTools (3.4.0)

Environment:
  OS: Linux archlinux 5.3.5-arch1-1-ARCH #1 SMP PREEMPT Mon Oct 7 19:03:08 UTC 2019 x86_64
  PHP Version: 7.3.10
  PHP SAPI: cli
  PHP Bin: /usr/bin/php
  PHP Extension Dir: /usr/lib/php/modules
  PHP Bin Dir: /usr/bin
  Loaded PHP config: /etc/php/php.ini
Versions:
  Phalcon DevTools Version: 3.4.0
  Phalcon Version: 3.4.4
  AdminLTE Version: 2.3.6

[user@archlinux phalcon]$ phalcon project qwerty

Phalcon DevTools (3.4.0)

PHP Warning:  mkdir(): Permission denied in /home/user/projects/php/phalcon-devtools/scripts/Phalcon/Builder/Project.php on line 105

  Error: Unable create project directory /qwerty/

**Comment by ruudboon** [comment]
@KozakIvan Looks like you're running Devtool 3.4.0. Can you upgrade to the latest (3.4.4) and try again

**Comment by KozakIvan** [comment]
[user@archlinux ~]$ cd ~/projects/php/phalcon-devtools/
[user@archlinux phalcon-devtools]$ git pull
Already up to date.
[user@archlinux phalcon-devtools]$ ./phalcon

Phalcon DevTools (3.4.0)

Available commands:
  info             (alias of: i)
  commands         (alias of: list, enumerate)
  controller       (alias of: create-controller)
  module           (alias of: create-module)
  model            (alias of: create-model)
  all-models       (alias of: create-all-models)
  project          (alias of: create-project)
  scaffold         (alias of: create-scaffold)
  migration        (alias of: create-migration)
  webtools         (alias of: create-webtools)
  serve            (alias of: server)
  console          (alias of: shell, psysh)

**Comment by KozakIvan** [comment]
when install latest 3.4.4 version from release page, ./phalcon show same 3.4.0 version

**Comment by ruudboon** [comment]
@KozakIvan Looks like default installation is root instead relative dir.
You can work around this issue by specifying the directory
```
./phalcon-devtools/phalcon project a123  --directory=/home/user/projects/a123
```

**Comment by ruudboon** [comment]
Fixed in 3.4.x branch. Thnx for reporting!
